### PR TITLE
Add last breadcrumb id to native report attributes

### PR DESCRIPTION
**Pull request by konraddysput**
# Why

This diff sets the last breadcrumb id attribute to native reports.

### CODE REVIEW

File: backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java [hunk 1]
```diff
@@ -308,6 +311,19 @@ public void onEvent(BacktraceResult backtraceResult) {
...
+    public void verifyBreadcrumbCallbackInvocation() {
+        backtraceClient.enableBreadcrumbs(context);
+        Breadcrumbs breadcrumbs = backtraceClient.database.getBreadcrumbs();
+
+        breadcrumbs.setOnSuccessfulBreadcrumbAddEventListener(breadcrumbId -> {
+            assertEquals(breadcrumbs.getCurrentBreadcrumbId(), breadcrumbId);
+            return;
+        });
+
+        breadcrumbs.addBreadcrumb("test");
```
Comment (BartoszLitwiniuk) [inline backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java:324]: are you sure that this assert is triggered?

Comment (konraddysput) [reply backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java:324]: yeah Im sure - it's a sync call. I also verified it in the debugger

**Review by BartoszLitwiniuk — APPROVED** [review]
lgtm

**Review by melekr — APPROVED** [review]
lgtm

**Pull request merged**